static/src/js/components/PublishPreview/PublishPreview.jsx

```diff
@@ -112,6 +113,15 @@ const PublishPreview = () => {
     ingestMutation(derivedConceptType, ummMetadata, nativeId, providerId)
   }
 
+  // Calls ingestDraft mutation with a new nativeId
+  const handleClone = () => {
+    const cloneNativeId = `MMT_${crypto.randomUUID()}`
+    // Removes the value from the metadata that has to be unique
+    removeMetadataKeys(ummMetadata, ['Name', 'LongName', 'ShortName'])
```

Comment:
Will this work for collection?   Does EntryTitle need to be added to this list?

Reply:
Yes this will work for Collections as well. The helper function takes in a list of keys and checks if the key exists. If it does it will remove it. So in this array I am just defining all the keys.

Reply:
I don't see EntryTitle as one of the keys?   Shouldn't this key be included?

Reply:
In current MMT, ShortName is the only field that is being removed when cloning a draft. I am guessing one ShortName has to be unique for Collections 
![image](https://github.com/nasa/mmt/assets/67551614/456f6908-6625-4c75-b5a1-de3a8070f3ec)